src/azure-cli/azure/cli/command_modules/cosmosdb/_params.py

```diff
@@ -47,6 +48,8 @@ def load_arguments(self, _):
     with self.argument_context('cosmosdb create') as c:
         c.argument('account_name', completer=None)
         c.argument('key_uri', help="The URI of the key vault", is_preview=True)
+        c.argument('enable_free_tier', arg_type=get_three_state_flag(), help="If enabled the account is free-tier.", is_preview=True)
```

Comment:
would the account be  created with various tiers besides free tier?

Reply:
Either it is FreeTier or it is not. No other related setting.

Reply:
Any further comments here ?